### ISSUE TITLE
[TextInputLayout] Issues with Talkback interaction

### ISSUE THREAD

**Issue by kenzieFlick** [material-components/material-components-android#604]
**Description:**  I have been running into issues surrounding Talkback accessibility and the error message with the TextInputLayout. When Talkback-testing the accessibility of TextInputLayout, which surrounds a TextInputEditText, the TextInputLayout is interacting unusually with Talkback announcements. It announces the validation error message set on the text field whenever a character is being typed or deleted regardless of if the error is active or not. 

**Expected behavior:** Using TextInputLayout in coordination with TextInputEditText should be announcing the character being typed and deleted in Talkback instead of announcing the error message of the text field. It should only announce the error message once on the text field when it is valid, not in multiple successions on any character addition or deletion.

**Screenshots:** 
Here it would read "invalid aisle" multiple times as I type or delete a character
<img width="328" alt="Screen Shot 2019-09-16 at 2 03 15 PM" src="https://user-images.githubusercontent.com/24287035/64985565-c3963b80-d88a-11e9-88af-5db112da6fe9.png">

**Source code:** The code snippet which is causing this issue. Please consider attaching a minimal sample app that reproduces the issue.
Layout being used:
`<com.google.android.material.textfield.TextInputLayout
        xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:app="http://schemas.android.com/apk/res-auto"
        android:id="@+id/single_text_input_layout"
        android:layout_height="wrap_content"
        android:layout_width="match_parent"
        app:helperTextEnabled="true"
        app:errorIconDrawable="@null"
        style="@style/textFieldLayout">

    <com.google.android.material.textfield.TextInputEditText
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:id="@+id/single_text_input_edit_text"
            style="@style/singleTextFieldEditText"/>
</com.google.android.material.textfield.TextInputLayout>`

**Android API version:** Marshmallow

**Material Library version:** 1.1.0-alpha10

**Device:** Device on which the bug was encountered here: Zebra TC51

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library. ✅ 

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by kenzieFlick** [comment]
Actually avoided the the error icon by adding `app:errorIconDrawable="@null"` to the TextInputLayout

**Comment by ldjcmu** [comment]
Thanks for this report. We'll look into reducing the error verbalization in the case you originally described.

**Comment by kenzieFlick** [comment]
@ldjcmu Thank you so much!

**Comment by drchen** [comment]
Is this still an issue?

**Comment by drchen** [comment]
I'll close this due to no further update. Feel free to reopen it if this is still an issue. : )

**Comment by harshfast** [comment]
Yes, this is still an issue, how to fix it. Error text is announced on every input.

**Comment by kthieuJHA** [comment]
This is still an issue on v1.12.0.

Even more apparent, for helper text in my scenario, talkback highlights the entire textinputlayout element but only reads out the helper text on first navigation, and continues to read out the helper text every time a character is added or deleted. 
Helper Text is "*required"
Talkback reads "Required for [textinputedittext text currently]"

In another scenario, I wrote code to handle custom accessibility delegates for the textinputlayout, but I am also unable to not make the helper text or error texts non-accessible as well; it makes a bad user experience when I have to swipe past it before going into the next element.
